extra-network/chisel/autobuild/build

```diff
@@ -0,0 +1,10 @@
+if [[ "${CROSS:-$ARCH}" != "loongson3" ]]; then
```

Comment:
Please use `if ! ab_match_arch loongson3; then` clause and add a comment explaining why.